string bot_name
time stamp 
---
bool success